linkerd/meshtls/util/Cargo.toml

```diff
@@ -1,11 +1,19 @@
 [package]
-name = "linkerd-meshtls-test-util"
+name = "linkerd-meshtls-util"
```

Comment:
nit, take it or leave it: i might have called this `linkerd-meshtls-core`, following the naming scheme we use elsewhere in the project?